### PR TITLE
Ref #8164 - Make user option optional in ssh-manager extension

### PR DESCRIPTION
**Pull request by yaronuliel**
## Description
Make the `user` option in the SSH Connection Manager optional (to fully support hosts defined in `~/.ssh/config`

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
Looks like a fine change for me, could you look at it @scopsy 

@yaronuliel do you mind adding a changelog entry

**Comment by yaronuliel** [comment]
@pernielsentikaer Added changelog

**Comment by pernielsentikaer** [comment]
Pinging @scopsy @LenoxFro @solid-pixel - can you check it out?

File: extensions/ssh-manager/src/create.tsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ export default function Main() {
     >
       <Form.TextField id="name" title="Connection Name" />
       <Form.TextField id="address" title="Server Address" placeholder={"A resolvable DNS name or IP"} />
-      <Form.TextField id="user" title="Username" placeholder={"A username to authenticate with"} />
+      <Form.TextField id="user" title="Username" placeholder={"A username to authenticate with (optional)"} />
```
Comment (solid-pixel) [inline extensions/ssh-manager/src/create.tsx:29]:
```suggestion
      <Form.TextField id="user" title="Username (optional)" placeholder={"A username to authenticate with (optional)"} />
```

**Review by solid-pixel — APPROVED** [review]
tested it locally and seems to work fine!
You only forgot to add `(optional)` next to the Username label :) I've added it.

**Review by pernielsentikaer — APPROVED** [review]

**Pull request merged**